src/tcp.c

```diff
@@ -450,6 +450,28 @@ tcp_get_ip_from_str(const char *src, struct sockaddr *sa)
 /**
  *
  */
+int
+tcp_get_sockaddr(struct sockaddr *sa, const char *s)
```

Comment:
Because I don't know about this other function... our patch is based on the work of Steffen Vogel.